zetta_utils/tensor_ops/common.py

```diff
@@ -388,7 +388,7 @@ def interpolate(  # pylint: disable=too-many-locals
         allow_slice_rounding=allow_slice_rounding,
     )
 
-    if mode == "segmentation" and (
+    if mode in ("segmentation", "img", "bilinear") and (
```

Comment:
Maybe also add `linear` and `trilinear`, just to prevent that the pytorch path is called for 3D volumes, which could be handled by tinybrain. Tinybrain will ignore it and do the downsampling based on the `scale_factor_tuple`.

Reply:
Addressed